arch/x86/core/intel64/cpu.c

```diff
@@ -26,8 +26,16 @@ extern u8_t _exception_stack1[];
 extern u8_t _exception_stack2[];
 extern u8_t _exception_stack3[];
 
+extern u8_t z_x86_trampoline_stack[];
+extern u8_t z_x86_trampoline_stack1[];
+extern u8_t z_x86_trampoline_stack2[];
+extern u8_t z_x86_trampoline_stack3[];
+
```

Comment:
Although this is not a syntax problem, put this declaration inside 
ifdef CONFIG_X86_KPTI would make it clear because it will be easy to understand
that these stacks are only used when KPTI is enabled.

Reply:
done